.github/labels.md

```diff
@@ -0,0 +1,252 @@
+# Issue and Pull Request labels
+
+This document describes the intended usage of GitHub Issue and Pull Request
+labels within the FreeOrion project.  In this document when referring to
+__Issues__ GitHub Issues are meant and when referring to __PR__ GitHub
+Pull Requests are meant.
+
+
+## Purpose of labels
+
+Labels allow attaching common search and filter terms to Issues/PRs.  This makes
+it easier for developers, managers and reporters to track the purpose, scope and
+state of Issues/PRs without explicitly reading through lengthy comments, code
+reviews or commits.  Please follow this guide as closely as possible to avoid
+duplicating work and wasting time of other developers and to enable better
+documentation on decisions made.
+
+
+## Label categories
+
+Labels are grouped in categories.  Those categories group together a common
+concept like the resolving state of an Issue/PR or the target platform where the
+Issue/PR occurs.  Categories share a prefix in the form
+`category_name:label_name` and are grouped together by color.  As a rule of
+thumb there is at least one label from every category that needs to be applied
+to an issue but use common sense to check if the label is appropriate or not for
+the specific Issue/PR.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe the intent of the Issue/PR.  Usually there
+should be exact one intent for creating an Issue/PR and it should not change
+during the lifetime of an Issue/PR.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue/PR affects or is
+affected by.  This should help developers to identify if they have required
+knowledge or interests to handle the issue.  There should be at least one
+component listed for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+Issue/PR have exactly one status.  Status may change over the lifetime of the
```

Comment:
Grammar: "An Issue/PR has exactly one status." or "Issues/PRs have exactly one status".

That said, something I missed when I did my first review: I'm not sure if requiring exactly one status strictly is going to work. Although that's probably accurate for 99% of all cases, e.g. I'd expect "status:testing requested" and "status:work in progress" might often go hand in hand.

So maybe say: "An Issue/PR usually has exactly one status."?